benji.py

```diff
@@ -47,7 +47,7 @@ def __init__(self):
      
         def OnEnter(self,event):
             put=self.txt.GetValue()
-            put=put.lower()
+            #put=put.lower()

```

Comment:
the string is converted to lower for the if else conditions. commenting out the line means 'SEARCH BATMAN' won't work like 'search batman' when it should

Reply:
but when searching for a file, filename is case sensitive, right?

Reply:
Depends on the method you are using. But a simple search on my file explorer finds the file in whichever case, lower or upper